recipe/meta.yaml

```diff
@@ -26,15 +26,14 @@ requirements:
     - setuptools
   run:
     - python
-    - ipykernel >=4.1  # not a real dependency, but require the reference kernel
+    - traitlets
```

Comment:
The alphabetical order of run dependencies (except python) will be more readable

Reply:
Great! Thanks